### DISCUSSION TITLE
test

### DISCUSSION THREAD

**Discussion by enihcam** [XTLS/Xray-core#259] [Ideas]
-

**Comment by RPRX** [comment]
此前已在研究通过 eBPF/XDP/SOCKMAP 进一步增强 XTLS 在路由器上的转发能力，希望它不像 Splice 一样受 IP 转发的影响。Splice 终究是在 Xray-core 的两个 TCP socket 间转发，若能将这一步移至 Kernel 网络栈之前、只简单修改包头，肯定能实现更强的性能。
